packages/logo/composer.json

```diff
@@ -2,6 +2,7 @@
 	"name": "automattic/jetpack-logo",
 	"description": "A logo for Jetpack",
 	"type": "library",
+	"version": "1.0.0",
```

Comment:
Note:

>When you add a hardcoded version to a VCS, the version will conflict with tag names. Composer will not be able to determine the version number.

via https://getcomposer.org/doc/02-libraries.md#library-versioning (edited)

and

> The version of the package. In most cases this is not required and should be omitted
>…
>Specifying the version yourself will most likely end up creating problems at some point due to human error. 

via 

https://getcomposer.org/doc/04-schema.md#version